pkg/test/masteruserrecord/assertion.go

```diff
@@ -102,6 +105,20 @@ func (a *Assertion) HasStatusUserAccounts(targetClusters ...string) *Assertion {
 	return a
 }
 
+func (a *Assertion) HasExpectedProvisionedTime(expectedProvisionedTime metav1.Time) *Assertion {
+	err := a.loadUaAssertion()
+	require.NoError(a.t, err)
+
+	if expectedProvisionedTime == (metav1.Time{}) {
```

Comment:
wouldn't be better to:
* either create another assertion method called `HasNoProvisionedTime()` instead of checking emptiness
* or accepting a pointer as a parameter `HasExpectedProvisionedTime(expectedProvisionedTime *metav1.Time)` when the MUR.status keeps the pointer instead of the value

Reply:
Good ideas! I went with creating two new functions, `verifyMurStatusExpectedProvisionedTime` and `verifyMurStatusNoProvisionedTime`

Reply:
Hm.. I don't see any updates in the PR but also would prefer a separate function here:
`HasNoProvisionedTime()` in addition to `ProvisionedAfter(expectedProvisionedTime metav1.Time)`